templates/field/field--paragraph--field-oe-text-long.html.twig

```diff
@@ -6,6 +6,6 @@
  * @see ./core/themes/stable/templates/field/field.html.twig
  */
 #}
-<div class="ecl-editor">
```

Comment:
This is a bit of a problem because oe-text-long is also used inside components (for example, inside the accordion item), which adds extra margins inside them, making them bigger than they should

Reply:
Good catch

Reply:
done